config/routes.rb

```diff
@@ -1,3 +1,8 @@
+# frozen_string_literal: true
+
 Rails.application.routes.draw do
+  root to: 'users#index'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  # resource :users, only: [:new, :create]
+  resources :users
```

Comment:
Use the` :only` option to generate the routes needed for your resource. See [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_route1.png) for new user routes and see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_route2.png) for edit user routes.